var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -599,6 +613,11 @@ def define_prefix_enable(prefix, cmake_var, spec_var=None):
                 define('Trilinos_ENABLE_SEACASNemslice', False)
             ])
 
+        if '+scorec' in spec:
+            options.extend([
+                define('Trilinos_ENABLE_SCOREC', True),
+            ])
+
```

Comment:
Can you roll this into the long list at line 579?
```python
            define_trilinos_enable('SCOREC'),
```

The "special cases" down at the bottom here are for having to define nonstandard options.

Reply:
Thanks for pointing that out, I missed that.